screen/contributors/src/main/java/com/ivy/contributors/ContributorsViewModel.kt

```diff
@@ -49,22 +49,18 @@ class ContributorsViewModel @Inject constructor(
     }
 
     private suspend fun fetchContributors() {
-        val contributors = ivyWalletRepositoryDataSource.fetchContributors()?.map {
+        val contributors = ivyWalletRepositoryDataSource.fetchContributors().map {
             Contributor(
-                name = it.login,
-                photoUrl = it.avatarUrl,
+                name = it.login ?: "Anonymous",
+                photoUrl = it.avatarUrl ?: "",
                 contributionsCount = it.contributions.toString(),
-                githubProfileUrl = it.link
+                githubProfileUrl = it.link ?: ""
             )
         }
 
-        if (contributors != null) {
-            contributorsResponse.value = ContributorsResponse.Success(
-                contributors.toImmutableList()
-            )
-        } else {
-            contributorsResponse.value = ContributorsResponse.Error("Error")
```

Comment:
Do we support an error case? What do we show if there's no internet?

Reply:
That's true. I missed that, sorry. I'll fix it.